### PR TITLE
Remove Wellness Check from Nav Header

### PR DESCRIPTION
**Pull request by bennettforkner**
This PR removes the wellness check from the Nav header as approved by Chris Imming. The URL `.../wellness` will still work as before, but it will be obfuscated and un-referenced.

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -210,7 +203,6 @@ const GordonHeader = ({ onDrawerToggle }) => {
               />
               {requiresAuthTab('People', <PeopleIcon />)}
               {/* {requiresAuthTab('Timesheets', WorkIcon)} */}
```
Comment (EjPlatzer) [inline src/components/Header/index.js:205]: Should we put Timesheets back in the header now that we're removing wellness?

Comment (bennettforkner) [reply src/components/Header/index.js:205]: I didn't know that was why it was removed. Yes! Let's add it back.
```suggestion
              {requiresAuthTab('Timesheets', WorkIcon)}
```

File: src/components/Header/index.js [hunk 2]
```diff
@@ -41,13 +40,7 @@ const GordonHeader = ({ onDrawerToggle }) => {
...
     // Tab url regular expressions must be listed in the same order as the tabs, since the
     // indices of the elements in the array on the next line are mapped to the indices of the tabs
-    let urls = [
-      /^\/$/,
-      /^\/involvements\/?$|^\/activity/,
-      /^\/events\/?$/,
-      /^\/people$/,
-      /^\/wellness$/,
-    ];
+    let urls = [/^\/$/, /^\/involvements\/?$|^\/activity/, /^\/events\/?$/, /^\/people$/];
```
Comment (EjPlatzer) [inline src/components/Header/index.js:43]: I think we need to add timesheets to the tab index.

**Review by EjPlatzer — APPROVED** [review]
LGTM!

**Pull request merged**